src/Yoti.Auth/Activity.cs

```diff
@@ -40,21 +40,54 @@ public ActivityDetails HandleSuccessfulResponse(AsymmetricCipherKeyPair keyPair,
             {
```

Comment:
The previous two statements both _return_, so we don't need this to be in an _else_ block

Reply:
Yep good policy, nice to get rid of noise when we can